src/en/community/events/2025/ceph-days-london/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Ceph Days London 2025
+date: 2025-06-04
+end: 2025-06-04
+location: London, United Kingdom
+venue: IBM UK, York Road
+image: "/assets/bitmaps/ceph-days.png"
+sponsors:
+tags:
+  - ceph days
+---
+
+### Bringing Ceph to London
```

Comment:
How about "Bringing Ceph to London yet again"?

Reply:
sure